core/src/main/scala/io/chrisdavenport/github/internals/RequestConstructor.scala

```diff
@@ -89,7 +89,7 @@ object RequestConstructor {
       .void
       .stream
 
-  private val GITHUB_URI: Uri = uri"https://api.github.com"
+  private val GITHUB_URI: Uri = uri"https://api.github.com/"
```

Comment:
Turns out this didn't work with relative urls anymore (probably due to the missing slash).
Adding the slash at the end fixes it.

Reply:
kk. Expect that will then be required as well for enterprise versions of that uri. Things to note when we get a bug report.

Reply:
Yes, in fact I remember I had to add the `/` at the end of the uri in the enterprise version to make it work.